javascript/ql/lib/change-notes/2022-01-08-insecure-dependency-resolution.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* The `js/http-dependency` query has been added. It detects depedencies that are downloaded using an unencrypted connection.
```

Comment:
The id in the ql file is `js/insecure-dependency`,
Not sure which id you want to use but both mentions (in the change notes and in the ql file) should state the same name.
```suggestion
* The `js/insecure-dependency` query has been added. It detects depedencies that are downloaded using an unencrypted connection.
```